### ISSUE TITLE
BatchNorm2d' object has no attribute 'num_batches_tracked'

### ISSUE THREAD

**Issue by diaodeyi** [uvipen/Yolo-v2-pytorch#19]
how to solve
